WcaOnRails/db/structure.sql

```diff
@@ -401,6 +402,22 @@ CREATE TABLE `Scrambles` (
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 --
+-- Table structure for table `ar_internal_metadata`
+--
+
+DROP TABLE IF EXISTS `ar_internal_metadata`;
```

Comment:
Forgot about this one!
These are rails internal stuff and I usually keep them out of the exported structure, should they be included in there?